vignettes/ci.Rmd

```diff
@@ -70,6 +77,27 @@ steps:
 
 See also the [example][github-actions-renv] on GitHub actions.
 
+### Simplify using a extenral helper Github action
+
+`r-lib` organization offers some actions for R users, and among them a [`setup-renv`][r-lib-actions-renv] action. To use this action, just add a `uses` step in your workflow: 
+
+```yaml
+steps:
+- uses: actions/checkout@v3
+- uses: r-lib/actions/setup-r@v2
+- uses: r-lib/actions/setup-renv@v2
+  with:
+    profile: '"shiny"'
```

Comment:
Is this necessary?

Reply:
No it is an example of the fact you can use profile feature. But let's leave that for the `setup-renv` documentation.